packages/wonder-blocks-link/components/link.js

```diff
@@ -89,6 +75,32 @@ export type SharedProps = {|
     onClick?: (e: SyntheticEvent<>) => void,
 |};
 
+export type SharedProps =
+    | BaseProps & {
+      /**
```

Comment:
Thanks for making these mutually exclusive.  Give that `BaseProps` is an exact object type, I think we should do the same here, e.g.
```
type SharedProps = {|
   ...BaseProps,
   skipClientNav?: boolean,
|} | {|
   ...BaseProps,
   openInNewTab?: boolean,
|};
```

Reply:
Oo! I like that better. Good call.